sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/query.sql

```diff
@@ -80,3 +80,5 @@ LEFT JOIN
   impressions
 USING
   (request_id)
+WHERE
```

Comment:
if we want to continue with this. i'm going to change this to `INNER JOIN` and drop the conditional stmt 🤦